shaders/post-processing/dof.frag.js

```diff
@@ -0,0 +1,63 @@
+module.exports = /* glsl */ `
+precision highp float;
+
+varying vec2 vTexCoord0;
+uniform sampler2D image; //Image to be processed 
+uniform sampler2D depthMap; //Linear depth, where 1.0 == far plane 
+
+uniform vec2 uPixelSize; //The size of a pixel: vec2(1.0/width, 1.0/height) 
+uniform float uFar; // Far plane  
+uniform float uNear;
+uniform float uFocusPoint;
+uniform float uFocusScale;
+
+const float GOLDEN_ANGLE = 2.39996323; 
+const float MAX_BLUR_SIZE = 20.0; 
+const float RAD_SCALE = 0.5; // Smaller = nicer blur, larger = faster
+
+
+float readDepth(const in sampler2D depthMap, const in vec2 coord, const in float near, const in float far) {
```

Comment:
Use the existing chunk here: https://github.com/pex-gl/pex-renderer/blob/master/shaders/chunks/depth-read.glsl.js